pom.xml

```diff
@@ -187,39 +197,21 @@
       </exclusions>
     </dependency>
 
-
     <dependency>
       <groupId>com.github.salesforce-marketingcloud</groupId>
       <artifactId>fuelsdk</artifactId>
       <version>${salesforce.marketing.cloud.version}</version>
     </dependency>
-
-    <dependency>
-      <groupId>org.apache.cxf</groupId>
-      <artifactId>cxf-rt-frontend-jaxws</artifactId>
-      <version>${cxf.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.cxf</groupId>
-      <artifactId>cxf-rt-bindings-soap</artifactId>
-      <version>${cxf.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.cxf</groupId>
-      <artifactId>cxf-rt-transports-http</artifactId>
-      <version>${cxf.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.cxf</groupId>
-      <artifactId>cxf-rt-ws-security</artifactId>
-      <version>${cxf.version}</version>
-    </dependency>
-
     <dependency>
       <groupId>commons-lang</groupId>
       <artifactId>commons-lang</artifactId>
       <version>2.6</version>
     </dependency>
+    <dependency>
```

Comment:
why is this being added?

Reply:
We were trying to rebuild the SFMC jar and remember adding few jar's during that time and this was one of them. I tried to check any dependency in the code for this library but could not find it and was able to build by removing the dependency.  Colgate has removed the my access to their environment so not able to confirm for sure by running the plugin(with out this jar). For now I can remove the dependency JAR and update the PR. Let me know.

Reply:
yes, please remove anything that's not needed, as it bloats the jar and can potentially cause conflicts later on

Reply:
Done